<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fafafa</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="mapa">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>431</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Pictures/Papel-de-parede-do-Spider-Man.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>441</width>
      <height>541</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buscaBT">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #00AFFA;
font: 12pt &quot;Arial&quot; bold;
color: white;
border: none;</string>
    </property>
    <property name="text">
     <string>BUSCAR</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="exibeBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>441</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="destCbBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="todoscaminhosCB">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Todos os caminhos</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="caminhosalterCB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Caminhos alternativos</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="menorCaminhoCB">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Melhor Caminho</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="origCbBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DIJSKTRA</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>59</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Origem</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Destino</string>
    </property>
   </widget>
   <widget class="QPushButton" name="matrizBT">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>530</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #00AFFA;
font: 12pt &quot;Arial&quot; bold;
color: white;
border: none;</string>
    </property>
    <property name="text">
     <string>Exibir Matriz </string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>mapa</zorder>
   <zorder>buscaBT</zorder>
   <zorder>exibeBrowser</zorder>
   <zorder>destCbBox</zorder>
   <zorder>todoscaminhosCB</zorder>
   <zorder>caminhosalterCB</zorder>
   <zorder>menorCaminhoCB</zorder>
   <zorder>origCbBox</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>matrizBT</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buscaBT</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>searchClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>searchClick()</slot>
 </slots>
</ui>
